ledger-core/virtual/object/object.go

```diff
@@ -220,6 +220,7 @@ type SMObject struct {
 	// dependencies
 	messageSender messageSenderAdapter.MessageSender
 	pulseSlot     *conveyor.PulseSlot
+	globalSem     conveyor.ParallelProcessingLimiter
```

Comment:
please, use the similar naming for all variables.`globalLimiter` or `globalSemaphore`

Reply:
done